yarn.lock

```diff
@@ -1,10786 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
[Please re-add this file](https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/).